api-server/web-server/src/api/json_helpers.rs

```diff
@@ -20,16 +20,16 @@ use common::{
 };
 use serde_json::json;
 
-pub fn amount_to_json(amount: Amount, chain_config: &ChainConfig) -> serde_json::Value {
-    json!(amount.into_fixedpoint_str(chain_config.coin_decimals()))
+pub fn amount_to_json(amount: Amount) -> serde_json::Value {
+    json!(amount.into_atoms().to_string())
```

Comment:
I think it could be changed like this:
```suggestion
    amount.into_atoms().to_string().into()
```
(Avoids the use of the macro, which should in general be preferred unless there's a cost in terms of verbosity or clarity.)

Reply:
fixed